### DISCUSSION TITLE
Hydra Migrate using TLS on Azure MySQL Flex Instance

### DISCUSSION THREAD

**Discussion by VirtualPenman** [ory/hydra#3638] [Q&A] [answered]
How can we enable the `hydra migrate sql` command to use TLS? I have an Azure MySQL Flex instance that requires TLS so each time I attempt to run the migration against the DB, I receive an error related to TLS being enforced from the server. 

```
Error 3159: Connections using insecure transport are prohibited while --require_secure_transport=ON.
```

In MySQL Workbench I just have SSL setting set to `true` and the CA file is set to the `DigiCertGlobalRootCA` provided by Microsoft.

**Comment by vinckr** [comment]
Hey @VirtualPenman 
First time I hear of this issue 🤔 I think there should be another solution for this. At least other users running on Azure have found one. 
So this should be solvable without adding extra code to Hydra! (Not sure how though sorry!)

**Reply by VirtualPenman** [reply]
Hey @vinckr  thanks for the reply. I ended up exporting a backup from an AKS DB to the MySQL Flex DB to get it all configured. But now I have Hydra running in a pod trying to connect to the MySQL DB and its receiving the same error as the `migrate` command. Is there a way to force Hydra in general to use TLS connections to the DB?

**Comment by VirtualPenman** [comment, marked as answer]
For anyone also facing an issue where Hydra is not using TLS to connect to the DB, I found this Hydra doc that details the param needed in the DSN connection string (for MySQL). The param is different than standard MySQL params. 

Add to end of DSN URL string `?tls=true`

[Hydra Doc](https://www.ory.sh/docs/self-hosted/deployment)
